### DISCUSSION TITLE
What is the status bar element supposed to do?

### DISCUSSION THREAD

**Discussion by swehba** [st3v3nmw/obsidian-spaced-repetition#597] [Q&A]
I see the following in the status bar:

![CleanShot 2023-01-10 at 11 05 57@2x](https://user-images.githubusercontent.com/1922612/211639698-839be0ae-8aa3-43c0-aaf7-dc94327b045a.jpg)

I assume this is related to the spaced repetition plugin. However, click on it just displays an empty popup that acts like a search field. See the following:

![CleanShot 2023-01-10 at 11 08 36@2x](https://user-images.githubusercontent.com/1922612/211639987-2ce4efda-5793-4f4f-9b44-9435766b2fc3.jpg)

What is it supposed to do? What is it searching for?

**Comment by ku-zhen** [comment]
> I see the following in the status bar:
> 
> ![CleanShot 2023-01-10 at 11 05 57@2x](https://user-images.githubusercontent.com/1922612/211639698-839be0ae-8aa3-43c0-aaf7-dc94327b045a.jpg)
> 
> I assume this is related to the spaced repetition plugin. However, click on it just displays an empty popup that acts like a search field. See the following:
> 
> ![CleanShot 2023-01-10 at 11 08 36@2x](https://user-images.githubusercontent.com/1922612/211639987-2ce4efda-5793-4f4f-9b44-9435766b2fc3.jpg)
> 
> What is it supposed to do? What is it searching for?

it's for note review, you should add some tags for your reviewing note in the plugin setting panel

**Reply by Ne110** [reply]
so,how to turn off the this display in the status bar?

**Comment by skorphil** [comment]
+1 to question. It seems like a bug and probably should show cards I need to review, but it shows nothing
